GovUk.Frontend.Umbraco.ExampleApp/uSync/v9/DataTypes/TPRNotificationBannerBlockList.config

```diff
@@ -0,0 +1,55 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
In this data type you are not configuring the label property. When you edit the data type and click on 'Button' or 'Link, styled as a button' it should have a custom label. Compare to the 'GOV.UK Block list' data type:

![Settings for including a 'Button' in 'GOV.UK Block list'](https://user-images.githubusercontent.com/1260550/215527319-de9245c6-5e28-4b40-890a-fd6ac1b835c3.png)

Following my other comments the name of this data type would become "GOV.UK Notification banner block list"